public/locale/en.json

```
@@ -1350,6 +1361,7 @@
   "first_name": "First Name",
   "food": "Food",
   "footer_body": "Open Healthcare Network is an open-source public utility designed by a multi-disciplinary team of innovators and volunteers. Open Healthcare Network CARE is a Digital Public Good recognised by the United Nations.",
+  "for_group": "(for the \"{{text}}\" Group)",
```

keep the brackets where it's called; i don't think this should be part of localizations

```suggestion
  "for_group": "for the \"{{text}}\" Group",
```